org.hlwd.biblemultithelife.yml

```diff
@@ -0,0 +1,51 @@
+app-id: org.hlwd.biblemultithelife
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: biblemultithelife
+separate-locales: false
+finish-args:
+  - --socket=x11
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=network
+  - --share=ipc
+  - --device=dri
+#  - --talk-name=org.freedesktop.portal.Desktop
+#  - --talk-name=org.freedesktop.portal.Documents
+  - --talk-name=org.freedesktop.portal.OpenURI
+  - --filesystem=xdg-run/xdg-desktop-portal
+#  - --filesystem=home
+#  - --filesystem=host
+modules:
+  - name: libkeybinder
+    buildsystem: autotools
+    config-opts:
+       - --disable-introspection
+    sources:
+       - type: archive
+         url: https://github.com/kupferlauncher/keybinder/releases/download/keybinder-3.0-v0.3.2/keybinder-3.0-0.3.2.tar.gz
+         sha256: e6e3de4e1f3b201814a956ab8f16dfc8a262db1937ff1eee4d855365398c6020
+  - name: biblemultithelife
+    buildsystem: simple
+    build-commands:
+       - mkdir -p /app
+       - tar -xf biblemultithelife.tar.gz -C /app
+       - mkdir -p /app/bin
+       - ln -s /app/biblemultithelife /app/bin/biblemultithelife
+       - ln -s /usr/lib/x86_64-linux-gnu/libsqlite3.so.0 /app/lib/libsqlite3.so
```

Comment:
how does this work on aarch64?

Reply:
I didn't test on aarch64. 
My target was for x64.

Reply:
but a aarch64 package is still built.

Reply:
I was not able to set: only-arches: [ 'x86_64' ], 
if you can give me a doc, I found nothing.

Reply:
I compiled for x86_64 by adding flathub.json :)